magda-web-client/src/Components/Dataset/Add/ErrorMessageBox.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { FunctionComponent } from "react";
+import { connect } from "react-redux";
+import { config } from "config";
+import ServerError from "./Errors/ServerError";
+
+type Props = {
+    error?: Error | null;
+};
+
+const ErrorMessageBox: FunctionComponent<Props> = props => {
+    const { error } = props;
+    if (!error) return null;
+    if (error instanceof ServerError) {
+        switch (error.statusCode) {
+            case 500:
+                return (
+                    <div className="au-body au-page-alerts au-page-alerts--error">
+                        <div>
+                            <span>
+                                Magda has encountered an error when submitting
+                                your dataset.
+                            </span>
+                        </div>
+                        <div>
+                            <span>
+                                Everything is still saved locally - please try
+                                again in 5 minutes.
+                            </span>
+                        </div>
+                    </div>
+                );
+            case 400:
+                return (
+                    <div className="au-body au-page-alerts au-page-alerts--error">
+                        <div>
+                            <span>
+                                Magda has encountered the following error:
```

Comment:
This needs a space before the error, and we should get rid of the full-stop at the end too:

<img width="873" alt="image" src="https://user-images.githubusercontent.com/900555/63488968-1d0d8380-c4f4-11e9-84ba-a5b14cf39e70.png">

Reply:
added a space between error: and actual error message and remove the extra `.`